Auth0/SafariWebAuth.swift

```diff
@@ -163,7 +163,7 @@ class SafariWebAuth: WebAuth {
             } else {
                 DispatchQueue.main.async {
                     guard let presenting = controller?.presentingViewController else {
-                        return callback(Result.failure(error: WebAuthError.cannotDismissWebAuthController))
```

Comment:
Maybe I'm missing something, why would this be nill? Do you have an example of use and debug.

Reply:
<img width="1072" alt="Screenshot 2019-10-29 at 3 24 10 PM" src="https://user-images.githubusercontent.com/26676738/67756552-37b62880-fa60-11e9-9814-0f4dd9cc43b9.png">
I debug in my project every time it will nil.

Reply:
Please can you provide a snippet and a way for me to be able to test this.  
Also this code has changed and is different (pending release), so please ensure steps work against master.

Reply:
I have checked in with the master branch same issue is reproducing.

Reply:
@ijknm I am unable to reproduce this without code samples from yourself and steps to reproduce this. In general it would be unusual for the controller to have vanished during authentication. If you are unable to produce these I will close this PR as I don't have a way to move this forward. Thanks